py-polars/tests/parametric/test_groupby_rolling.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+
+from datetime import timedelta
+from typing import TYPE_CHECKING
+
+import hypothesis.strategies as st
+from hypothesis import given, reject
+
+import polars as pl
+from polars.testing import assert_frame_equal
+from polars.testing.parametric.primitives import column, dataframes
+from polars.testing.parametric.strategies import strategy_closed, strategy_time_unit
+from polars.utils.convert import _timedelta_to_pl_duration
+
+if TYPE_CHECKING:
+    from polars.type_aliases import ClosedInterval, TimeUnit
+
+
+@given(
+    period=st.timedeltas(min_value=timedelta(microseconds=0)),
+    offset=st.timedeltas(),
```

Comment:
FYI Hypothesis has a handy [`map()` method for strategies](https://hypothesis.readthedocs.io/en/latest/data.html#mapping). Current the type hints of `period`/`offset` are wrong in the signature anywho.

```suggestion
    offset=st.timedeltas().map(_timedelta_to_pl_duration)
```

(had this thought and was going to message you on slack, but thought I'd check open PRs first :sweat_smile:)

Reply:
nice, good one, thanks!